### ISSUE TITLE
[Bug] temporal modules options are messed up

### ISSUE THREAD

**Issue by veroandreo** [OSGeo/grass#1653]
Options of temporal modules got messed up somehow in the current master branch.

To reproduce:
- update to latest master branch and just try:
```
t.create --h
Creates a space time dataset.

Usage:
 t.create inputs=name[,name,...] maps=name[,name,...] [type=name]
   semantictype=string title=string description=string [--help]
   [--verbose] [--quiet] [--ui]

Parameters:
        inputs   Name of the input space time datasets <<<<-- !!!????
          maps   Type of the output space time dataset <<<<-- !!!????
          type   Type of the input map                            <<<<-- !!!????
                 options: raster,vector,raster_3d
                 default: raster
  semantictype   Semantic type of the space time dataset
                 options: min,max,sum,mean
                 default: mean
         title   Title of the new space time dataset
   description   Description of the new space time dataset


t.register --h
Assigns timestamps and registers raster, vector and raster3d maps in a space time dataset.

Usage:
 t.register [-i] [nprocs=value] [map=name] [type=name] [file=name]
   [start=string] [end=string] [unit=string] [increment=string]
   [separator=character] [--overwrite] [--help] [--verbose] [--quiet]
   [--ui]

Flags:
  -i   Create an interval (start and end time) in case an increment and the start time are provided

Parameters:
     nprocs   Number of threads for parallel computing <<<<-- !!!????
              default: 1
        map   Name of the input map
       type   Type of the input space time dataset
              options: strds,stvds,str3ds
              default: strds
       file   Input file with map names, one per line
      start   Valid start date and time of the first map
        end   Valid end date and time of all map
       unit   Time stamp unit
              options: years,months,days,hours,minutes,seconds
  increment   Time increment, works only in conjunction with start option
  separator   Field separator character of the input file
              default: pipe
```
**Expected behavior**
Get the options right. The source code of the modules is just fine, so the mess comes from somewhere else.

**System description:**
- Operating System: Linux Fedora 33
```
GRASS nc_basic_spm_grass7/user1:grass79_dev > g.version -rge
version=8.0.dev
date=2021
revision=c48a2dd35
build_date=2021-06-17
build_platform=x86_64-pc-linux-gnu
build_off_t_size=8
libgis_revision=d2a5e8e8f
libgis_date=2021-06-16T04:05:21+00:00
proj=6.3.2
gdal=3.1.4
geos=3.8.1
sqlite=3.34.1

python3 -c "import sys, wx; print(sys.version); print(wx.version())"
3.9.5 (default, May 14 2021, 00:00:00) 
[GCC 10.3.1 20210422 (Red Hat 10.3.1-1)]
4.0.7 gtk3 (phoenix) wxWidgets 3.0.5
```

**Additional context**
I discovered this while trying to test #1650. I have used the temporal framework with master branch a couple of months ago and all was fine.

**Comment by wenzeslaus** [comment]
Caused by #1644, discovered by tests (thank you tests!), fixed in #1652.

**Comment by veroandreo** [comment]
I confirm it's fixed now, closing
